index.html

```diff
@@ -197,5 +210,36 @@ <h3>Skills</h3>
     <p>myemailhere@email.com</p>
 </div>
 </section>
+<section class="form-background">
+    <div class="form-flex-d">
+    <p class="text-contact">I'm always interested 
+        in hearing about new projects, so if you'd 
+        like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/mnqynkkd" method="post" class="form">
+        <label for="Full name"></label>
+        <div class="div-name">
+        <input class="inp-name" id="Full-name" placeholder="First Name" type="text" maxlength="30" required><input class="inp-name" id="last-name" placeholder="Last Name" type="text" maxlength="30" required></div>
```

Comment:
- [ ] Great job on meeting all other project requirements 👏 . However please I kindly suggest that you have just one input for "Full Name"(just like the mobile version), and this should take the space of both First Name and Last Name inputs.